src/app/app-modules/core/components/app-header/app-header.component.ts

```
@@ -111,6 +119,26 @@
     }
   }
 
+  refreshLogin(){
+    this.auth.getUserDetails().subscribe((res: any) => {
+      // console.log(">>>>>>", res);
+      if (res.statusCode == '200') {
+        if (res.data.previlegeObj && res.data.previlegeObj[0]) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)